### PR TITLE
feat: add article for Gradle support in Hilla

### PR DESCRIPTION
**Pull request by taefi**
Fixes: #2370

### CODE REVIEW

File: articles/lit/start/gradle.adoc [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+
+apply plugin: 'dev.hilla' // this is how it works in case of using pre-releases
+----
+
+- Add `buildscript` to the [filename]#settings.gradle# file containing the followings:
+
+.[filename]#settings.gradle# file might not exist in your project
+[NOTE]
+The [filename]#settings.gradle# file is mostly used within multi-module projects, but it's useful for other configurations.
+Thus, if you don't already have it in your project, create a plain text file called `settings.gradle` next to your [filename]#build.gradle# file, which is normally in the project root folder.
```
Comment (cromoteca) [inline articles/lit/start/gradle.adoc:443]:
```suggestion
Thus, if you don't already have it in your project, create a plain text file called [filename]`settings.gradle` next to your [filename]`build.gradle` file, which is normally in the project root folder.
```

File: articles/lit/start/gradle.adoc [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+  //... other plugins
+  id 'war'
+}
+
+war {
+    enabled = true
+}
+----
+
+- Your application class that is annotated with `@SpringBootApplication` should extend [classname]`SpringBootServletInitializer` and override the [methodname]`configure()` method:
```
Comment (cromoteca) [inline articles/lit/start/gradle.adoc:368]:
```suggestion
- Your application class that is annotated with [annotationname]`@SpringBootApplication` should extend [classname]`SpringBootServletInitializer` and override the [methodname]`configure()` method:
```

**Review by cromoteca — CHANGES_REQUESTED** [review]
Huge effort! I made many comments, but most, if not all, of them, are just trivial syntax fixes.

File: articles/lit/start/gradle.adoc [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+If you plan to use Hilla's Gradle-based starter projects, there is no need to install Gradle on your machine.
+A Gradle Wrapper script is included in those starter projects.
+This script manages the download and execution of Gradle locally for your project.
+For more information on using Gradle Wrapper, please refer to the https://docs.gradle.org/current/userguide/gradle_wrapper.html[Official Gradle Documentation].
+If you prefer to install Gradle directly, you can find the instructions at https://gradle.org/install
+
+A Hilla starter project that uses Gradle as the build and dependency management tool can be obtained from:
+
+1. https://start.spring.io/[Spring Initializr] which enables you to configure a bare-bones Hilla project by adding the `Hilla` dependency
+2. Directly downloading a
```
Comment (platosha) [inline articles/lit/start/gradle.adoc:61]: I think this should be preferred, it does contain initial frontend at least.

**Review by platosha — APPROVED** [review]
The amount of features covered is really good here. I have just one minor suggestion, otherwise LGTM.

**Review by Mikaelsu — APPROVED** [review]

**Pull request merged**